whitepapers/0011_accountant.md

```diff
@@ -0,0 +1,190 @@
+# Global Accountant
+
+This document describes the motivation, goals, and technical design of the "Global Accountant".
+
+The Global Accountant is a safety feature for registered token bridges.  It is implemented as an [Integrity Checker](0010_integrity_checkers.md) on Wormchain.
+
+## Objectives
+
+Tightly limit the impact an exploit in a connected chain can have on a registered token bridge.  Ensure the number of wrapped tokens capable of being moved out of a chain never exceed the number sent to that chain, even if that chain was compromised.
+
+## Background
+
+Token Bridges built on Wormhole (e.g. [Portal](0003_token_bridge.md)) allow transferring tokens between any of the various blockchains connected to the bridge.
+
+A core feature of the token bridge is that xAssets are fungible across all chains.  ETH sent from Ethereum to Solana to Polygon is the same as the ETH sent from Ethereum to Aptos to Polygon.  Transfers work by locking native assets (or burning wrapped assets) on the source chain and emitting a Wormhole message that allows the minting of wrapped assets (or unlocking of native assets) on the destination chain.
+
+It is unfortunately not practical to synchronize the state of the token bridge across all chains: If a transfer of Portal-wrapped ETH happens from Solana to Polygon, the token bridge contract on Ethereum is not aware of that transfer and therefore doesn't know where the wrapped assets are currently located.
+
+If any connected chain gets compromised, e.g. through a bug in the rpc nodes, this could cause the arbitrarily mint of “unbacked” wrapped assets, i.e. wrapped assets that are not backed by native assets.  Those unbacked wrapped assets could then be transferred to the native chain.  Because wrapped assets are fungible, the smart contract on the native chain cannot distinguish those unbacked wrapped assets and would proceed with unlocking the native assets, effectively draining the bridge.
```

Comment:
Nit: "arbitrary minting" or "arbitrary mint" rather than "arbitrarily mint"

Reply:
Fixed in ff42eb2b1b9651e4036d6d48ae3def0570bfcd3d